WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -874,6 +874,23 @@ private boolean checkNetworkConnectivity() {
         return true;
     }
 
+    private boolean isUserAlreadyLoggedIn() {
```

Comment:
I think we should either call this method `checkIsUserAlreadyLoggedIn` or handle all of the toast stuff inside `signIn`. Otherwise the method does more than what it's supposed to and if it's used somewhere else, it'll create an unwanted toast.

If we go with the 2nd option, we can also change the check in `fetchAccountSettingsAndSites` to use this method.